CorsixTH/Lua/world.lua

```diff
@@ -2365,6 +2366,44 @@ function World:resetAnimations()
   end
 end
 
+strict_declare_global "staff_initials_cache"
+staff_initials_cache = {}
+
+local function our_concat(t)
+  -- The standard table.concat function doesn't like our userdata strings :(
+  local result = ""
+  for _, s in ipairs(t) do
+    result = result .. s
+  end
+  return result
+end
+
+function World:updateInitialsCache()
+  -- Generate new set of staff name initials in case the language was changed
```

Comment:
This "in case language changed" check isn't done here. Plus it doesn't reflect the function's use in World:World.

Reply:
Corrected the comments.